core/src/main/java/org/hibernate/ogm/type/impl/StringTimestampTypeDescriptor.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.type.impl;
+
+import java.sql.Timestamp;
+import java.util.Date;
+
+import org.hibernate.MappingException;
+import org.hibernate.engine.spi.Mapping;
+import org.hibernate.ogm.type.descriptor.impl.TimestampDateTypeDescriptor;
+import org.hibernate.ogm.type.descriptor.impl.TimestampMappedAsStringGridTypeDescriptor;
+
+/**
+ * For {@link Timestamp} objects use a String representation.
+ *
+ * @author Davide D'Alto
+ */
+public class StringTimestampTypeDescriptor extends AbstractGenericBasicType<Date> {
```

Comment:
Serializing a Date as a String: maybe the actual String written on datastore could be TimeZone dependent.
What could happen if the server, who did the put operation, had a different TimeZone from the server that perform a get on the same Entity? I'm asking myself if the same Date would be return in that case.

Reply:
Yes, that it is an issue right now, when I worked on that I didn't consider timezones. We have actually a conversion for dates that comes with the class Iso8601StringDateType. I would like to use these types instead.

We need to be careful though because it will change the mapping. We should create a separate issue.

Good quesiton

Reply:
I see the point here. Maybe in another issue we could deep the topic.